test/testQMLFile.js

```diff
@@ -1229,5 +1230,111 @@ module.exports.qmlfile = {
         test.equal(r.getKey(), "16: disambiguation string");
 
         test.done();
-    }
+    },
+    testQMLPseudoLocalization1: function(test) {
+        test.expect(6);
+
+        var qf = new QMLFile({
+            project: p,
+            pathName: "./t1.qml",
+            type: qmlft
+        });
+        test.ok(qf);
+
+        qf.extract();
+        var set = qf.getTranslationSet();
+        test.equal(set.size(), 8);
+
+        var r = set.get(ContextResourceString.hashKey("app", "t1", set.sourceLocale, "Invalid Format", "x-qml"));
+        test.ok(r);
+
+        test.equal(r.getSource(), "Invalid Format");
+        test.equal(r.getKey(), "Invalid Format");
+
+        var rb = new RegularPseudo();
```

Comment:
Needs a "type" property in the argument to RegularPseudo.

Reply:
If it's not written, It will be set as default value `text`https://github.com/iLib-js/loctool/blob/development/lib/RegularPseudo.js#L45

Reply:
line 45 will throw an exception because `options.type` is trying to dereference `undefined`

Reply:
I added `type` property to test cases to make sure.  my last PR to loctool, also included guard code to prevent that case. (https://github.com/iLib-js/loctool/pull/86/commits/52f546b867470b0afc944e42f287eba1b26a6543)